orttraining/pytorch_frontend_examples/nvidia-bert/NOTICE

```diff
@@ -0,0 +1,5 @@
+BERT PyTorch
```

Comment:
This file could also be replaced with a pointer or with content that is more applicable for this repo.

Reply:
From [section 4.d](https://www.apache.org/licenses/LICENSE-2.0): If the Work includes a "NOTICE" text file as part of its distribution, then any Derivative Works that You distribute must include a readable copy of the attribution notices contained within such NOTICE file, excluding those notices that do not pertain to any part of the Derivative Works, in at least one of the following places: within a NOTICE text file distributed as part of the Derivative Works; within the Source form or documentation, if provided along with the Derivative Works; or, within a display generated by the Derivative Works, if and wherever such third-party notices normally appear. The contents of the NOTICE file are for informational purposes only and do not modify the License. You may add Your own attribution notices within Derivative Works that You distribute, alongside or as an addendum to the NOTICE text from the Work, provided that such additional attribution notices cannot be construed as modifying the License.